README.md

```diff
@@ -1,5 +1,21 @@
 # Adobe Experience Platform - Test App for Cordova Plugins
 
+## Notice of deprecation
```

Comment:
nit: Should we add ⚠️ badge to highlight the deprecation message?

## ⚠️  Notice of deprecation